en/development/configuration.rst

```diff
@@ -282,6 +282,19 @@ data back::
 
 If $key is left null, all values in Configure will be returned.
 
+.. php:staticmethod:: readOrFail($key)
+
+Reads configuration data just php:staticmethod:: read but expects to find a key/value pair.
```

Comment:
The `php:staticmethod:: read` defines the method link target and formats the method heading. The code you originally had looked like a 'link', but id now a method definition which is fine.